Vagrantfile

```diff
@@ -46,120 +47,156 @@ vegastrike_assets_repository = "https://github.com/vegastrike/Assets-Production.
+            vs_ubuntu_latest.vm.network "private_network", ip: "#{base_network_vb}.10"
+            vs_ubuntu_latest.vm.hostname = "vegastrike-ubuntu-latest"
+            vs_ubuntu_latest.vm.boot_timeout = 900
+            vs_ubuntu_latest.vm.provider "virtualbox" do |vb|
+                vb.memory = "#{base_memory}"
+                vb.cpus = "#{base_cpu_count}"
+                vb.customize ["modifyvm", :id, "--ostype", "Ubuntu_64"]
+                vb.customize ["modifyvm", :id, "--hwvirtex", "on"]
+                vb.customize ["modifyvm", :id, "--pae", "on"]
+                vb.customize ["modifyvm", :id, "--nestedpaging", "on"]
+                vb.customize ["modifyvm", :id, "--vram", "#{base_vram}"]
+                vb.customize ["modifyvm", :id, "--accelerate3d", "on"]
+                vb.customize ["modifyvm", :id, "--accelerate2dvideo", "on"]
+                vb.customize ["modifyvm", :id, "--audioout", "on"]
+                vb.gui = true
+            end
+            vs_ubuntu_latest.vm.provision "shell", inline: <<-SHELL
+                apt-get update
+                apt-get install -y ansible python-apt
+                apt-get install -y git
+                apt-get install -y "#{debian_desktop_environment}" virtualbox-guest-dkms virtualbox-guest-utils virtualbox-guest-x11
+                pushd /home/vagrant
+                if [ ! -d "Assets-Production" ]; then
+                su -c "git clone #{vegastrike_assets_repository}" vagrant
+                else
+                pushd "Assets-Production"
+                su -c "git pull" vagrant
+                popd
+                fi
+                popd
+                reboot
+            SHELL
+        end
+    end
 
-	if system_status_focal then
-		config.vm.define "vegastrike_ubuntu_focal" do | vs_ubuntu_focal |
-			vs_ubuntu_focal.vm.box = "#{image_ubuntu_focal}"
-			vs_ubuntu_focal.vm.network "private_network", ip: "#{base_network_vb}.11"
-			vs_ubuntu_focal.vm.hostname = "vegastrike-ubuntu-focal"
-			vs_ubuntu_focal.vm.provider "virtualbox" do |vb|
-				vb.memory = "#{base_memory}"
-				vb.cpus = "#{base_cpu_count}"
-				vb.gui = true
-			end
-			vs_ubuntu_focal.vm.provision "shell", inline: <<-SHELL
-				apt-get update
-				apt-get install -y ansible python-apt
-				apt-get install -y git
-				apt-get install -y "#{debian_desktop_environment}"
-				pushd /home/vagrant
-				if [ ! -d "Assets-Production" ]; then
-				su -c "git clone #{vegastrike_assets_repository}" vagrant
-				else
-				pushd "Assets-Production"
-				su -c "git pull"
-				popd
-				fi
-				popd
-				reboot
-			SHELL
-		end
-	end
+    if system_status_focal then
+        config.vm.define "vegastrike_ubuntu_focal" do | vs_ubuntu_focal |
+            vs_ubuntu_focal.vm.box = "#{image_ubuntu_focal}"
+            vs_ubuntu_focal.vm.network "private_network", ip: "#{base_network_vb}.11"
+            vs_ubuntu_focal.vm.hostname = "vegastrike-ubuntu-focal"
+            vs_ubuntu_focal.vm.boot_timeout = 900
```

Comment:
Would be good to make that a variable at the top.

Reply:
Good point.